src/main/java/htsjdk/samtools/util/BlockCompressedOutputStream.java

```diff
@@ -307,6 +307,19 @@ public void write(final byte[] bytes, int startIndex, int numBytes) throws IOExc
         }
     }
 
+    /**
+     * Writes the specified byte to this output stream. The general contract for write is that one byte is written
+     * to the output stream. The byte to be written is the eight low-order bits of the argument b.
+     * The 24 high-order bits of b are ignored.
+     * @param bite
+     * @throws IOException
+     */
+    @Override
+    public void write(final int bite) throws IOException {
```

Comment:
```suggestion
    public void write(final int b) throws IOException {
```

To match java doc. Alternatively, switch `b` to `bite` in docs.